src/MessageConsumer.cs

```diff
@@ -1251,15 +1251,11 @@ public virtual async Task AfterMessageIsConsumedAsync(MessageDispatch dispatch,
                 }
                 else if(IsClientAcknowledge || IsIndividualAcknowledge)
                 {
-                    bool messageAckedByConsumer = false;
 
                     using(await this.deliveredMessagesLock.LockAsync().Await())
                     {
-                        messageAckedByConsumer = this.deliveredMessages.Contains(dispatch);
-                    }
+                        if (this.deliveredMessages.Contains(dispatch))
```

Comment:
There is something messed up with the indentation here. Please add the parentheses right below the 'if' so the intention is clear.

Reply:
fixed